### DISCUSSION TITLE
Please adapt to win11

### DISCUSSION THREAD

**Discussion by ZYALBERT** [File-New-Project/EarTrumpet#1055] [Q&A] [answered]
Because win11 cannot close the volume icon, it is easy to click the wrong icon. Can it be integrated into the system volume icon control bar? 
![QQ截图20220207223223](https://user-images.githubusercontent.com/45811850/152807751-784be0e1-c216-4d68-ad86-9fc35a113d1f.png)

**Comment by riverar** [comment, marked as answer]
Unfortunately, Windows 11 does not allow for customization of this area. Please let them know of your disappointment via Feedback Hub!

You can switch to our legacy icon via Settings.

We are thinking about other alternatives as well.

**Comment by lichtmetzger** [comment]
You can run ExplorerPatcher though to switch back to the Windows 10 style taskbar.

With ExplorerPatcher active, run Win+R and enter this command:
shell:::{05d7b0f4-2121-4eff-bf6b-ed3f69b894d9}

This will open the old dialog where you can turn off system symbols. Disable the volume icon here - tadaa, now you only have EarTrumpet in your taskbar. :)

**Comment by D-Arora** [comment]
If you get [StartAllBack](https://www.startallback.com/) you can keep the Win11 taskbar but modify the notification area icons that show.

![image](https://user-images.githubusercontent.com/67812826/156465339-e43ecaa2-28c9-49b0-a0ff-deaaca87f9a5.png)
